Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -45,8 +48,33 @@ ABenchmarkGymGameMode::ABenchmarkGymGameMode()
 
 	NPCSToSpawn = 0;
 	SecondsTillPlayerCheck = 15.0f * 60.0f;
+}
 
-	RNG.Initialize(123456); // Ensure we can do deterministic runs
+void ABenchmarkGymGameMode::GenerateTestScenarioLocations()
+{
+	const float ROAM_RADIUS = 7500.0f;
```

Comment:
No need for caps, and could you add a comment how we get to this value (half actor ncd)